torntools/scripts/content/faction/ttFaction.js

```diff
@@ -1780,3 +1781,11 @@ function exportChallengeContributionsCSV() {
 		contributionsObserver.observe(doc.find("div#factions div#faction-upgrades div.body div#stu-confirmation"), { childList: true, subtree: true });
 	});
 }
+
+function addNumbersToMembers() {
+	let memberIndex = 1;
+	doc.findAll("div#faction-info-members ul.table-body li.table-row").forEach((memberRow) => {
```

Comment:
`(row, index) =>` that way you don't have to update the index yourself, just show `index + 1` then.

Reply:
Done.